### ISSUE TITLE
Video tutorial

### ISSUE THREAD

**Issue by Codrin-Popa** [CosyClub/ProjectTempo#139]
It would be nice to have a video tutorial of the game. It would show pre-recorded gameplay with text (and maybe audio) instructions.

**Comment by grant-m-s** [comment]
I don't think this is a great idea. The 'feeder areas' should work as 'learn as you play'. If the player seems to be stuck then I feel that #56 would be a nicer way to give hints.
